Source/Order.swift

```diff
@@ -54,16 +89,49 @@ final public class Order: NSObject, Codable {
                   smallest decimal unit of the currency (e.g. 3999 for $39.99).
         - currencyCode: The ISO 4217 currency code (default is USD).
      */
+    @available(*, deprecated, message: "No longer supported")
     @objc public init(id: String, amount: Int64 = 0, currencyCode: String = "USD") {
         self.id = id
         self.amount = amount
         self.currencyCode = currencyCode
+        self.purchaseDate = nil
+        self.customerOrderId = nil
+        self.lineItems = nil
+        self.customer = nil
+    }
+    
+    /**
+     Initializes an order object with the passed parameters.
+     
+     - Parameters:
+     - id: The order identifier (required).
+     - amount: The total order value in pennies or the
+               smallest decimal unit of the currency (e.g. 3999 for $39.99).
+     - currencyCode: The ISO 4217 currency code (default is USD).
+     - purchaseDate: The date of the purchase for the order.
+     - customerOrderId: The customer-face order Id.
+     - lineItems: A list of the line item details that comprise the order.
+     - customer: Customer field
+     */
+    @objc public init(id: String, currencyCode: String = "USD", purchaseDate: Date,
```

Comment:
> currencyCode: String = "USD"

How does this default value work for Objective-C consumers? It should not be part of the initializer for Objective-C consumers.

Reply:
Same question for the rest of the optional parameters.